src/walletSdk/Types/horizon.ts

```diff
@@ -12,3 +12,14 @@ export type TransactionParams = {
   memo?: Memo;
   timebounds?: Server.Timebounds | number;
 };
+
+export type SubmitWithFeeIncreaseParams = {
+  sourceAddress: SigningKeypair;
```

Comment:
Why is it SigningKeyPair? It should be just a regular AccountKeyPair.
We can create a transaction without owning a signing key, send it to the server to sign, and submit. 
It makes sense to do it on the device if the transaction manages device's account, but it's owned by custodial service (for example)

Reply:
👍  https://github.com/stellar/typescript-wallet-sdk/pull/31/commits/aa70c042ae7b62225fac9122e83a9000c159aac5